src/strings/PowerFxResources.en-US.resx

```diff
@@ -1917,6 +1917,10 @@
     <value>'{0}' is an unknown or unsupported function.</value>
     <comment>Error Message.</comment>
   </data>
+  <data name="ErrknownTypeHelperFunction" xml:space="preserve">
+    <value>'{0}' is recognized as type helper function and it can only be used within a type literal.</value>
```

Comment:
>literal [](http://example.com/codeflow?start=93&length=7)

Should this be type function? Not sure type literal makes sense here.

Reply:
confirmed with Greg on this and will fix this instance in this PR and others in a small follow up PR